### DISCUSSION TITLE
Writing File in VB

### DISCUSSION THREAD

**Discussion by theBest3000** [Apollo3zehn/PureHDF#122] [Q&A] [answered]
Hi.
I love your project. I have however some trouble with writing files in visual basic.
Even something like

Dim NewFile As New H5File
NewFile.Write("Path")

does not work and result in a corrupted file.

Can I not use NewFile.Add(H5Group) and MyGroup.Add(H5Attribute)?

Thank you in advance!

**Comment by Apollo3zehn** [comment, marked as answer]
I did a quick test and it worked for me. But I suspect you have a different setup ... maybe a different framework?

![grafik](https://github.com/Apollo3zehn/PureHDF/assets/20972129/62c7a1ce-3095-4ab9-9a45-c4ac5267c1e6)

**Comment by theBest3000** [comment]
Thank you. I tried to open the file with HDF Explorer. Apperently, this software is outdated and can't open the file.
